examples/adapterdrop/drop_at_inference.py

```diff
@@ -0,0 +1,25 @@
+import torch
```

Comment:
@arueckle Wasn't sure what to do with this file. Should we keep (and maybe extend) it or better move the content to the docs or notebooks?

Reply:
We can keep it for now and delete it as soon as we have a colab notebook.